instagram/src/components/Post/Post.js

```diff
@@ -0,0 +1,58 @@
+import React from "react";
+import CommentSection from "../CommentSection/CommentSection";
+import PropTypes from "prop-types";
+
+const Post = ({
+  post,
+  toggleLike,
+  newCommentText,
+  changeHandlerNested,
+  addNewComment
+}) => {
```

Comment:
very nice use of destructured props to explicitly state what you need access to in the component, as well as reference for yourself what you haven't utilized yet